src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1065,6 +1065,29 @@ def test_functionapp_consumption_e2e(self, resource_group, storage_account):
 
         self.cmd('functionapp delete -g {} -n {}'.format(resource_group, functionapp_name))
 
+    @ResourceGroupPreparer(name_prefix='azurecli-functionapp-c-e2e-ragrs', location='westus')
+    @StorageAccountPreparer(sku='Standard_RAGRS')
+    def test_functionapp_e2e_ragrs_storage(self, resource_group, storage_account):
+        functionapp_name = self.create_random_name('functionappconsumption', 40)
+
+        self.cmd('functionapp create -g {} -n {} -c westus -s {}'
+                 .format(resource_group, functionapp_name, storage_account)).assert_with_checks([
+                     JMESPathCheck('state', 'Running'),
+                     JMESPathCheck('name', functionapp_name),
+                     JMESPathCheck('hostNames[0]', functionapp_name + '.azurewebsites.net')])
+
+        self.cmd('functionapp list -g {}'.format(resource_group), checks=[
```

Comment:
I believe you can remove this and the following 2 test scenarios which are pretty much redundant testing coverage

Reply:
@yugangw-msft Thanks. I simplify the test case but I think this test is essential. It tests the function app creation against Standard_RAGRS storage account type specifically. Others are testing against Standard_LRS.